docs/conf.py

```diff
@@ -52,3 +52,4 @@
 # -------------------------------------------------------------------------
 default_role = "any"
 nitpicky = True
+suppress_warnings = ["myst.xref_missing"]
```

Comment:
Is it possible not to ignore this? (also a separate PR candidate)

Reply:
It'd be needed to create [cross-reference targets](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html) for all the detached links and images.

Reply:
How so?

Reply:
It's fixed here, but there will be some extra work once I convert the index in the other PR. Let's merge this first?